numpy/core/src/umath/loops_modulo.dispatch.c

```diff
@@ -0,0 +1,5714 @@
+#line 1 "numpy/core/src/umath/loops_modulo.dispatch.c.src"
+
+/*
+ *****************************************************************************
+ **       This file was autogenerated from a template  DO NOT EDIT!!!!      **
+ **       Changes should be made to the original source (.src) file         **
+ *****************************************************************************
+ */
```

Comment:
Seems there is an auto generated file here, can you remove it?

Reply:
Hello @BvB93, thanks for your reply. We removed this file in the recent commits...